File: sample-videochat-webrtc/sample-videochat-webrtc/Components/Settings/VideoSettingsViewController/BaseItemModel.m
//
//  BaseItemModel.m
//  sample-videochat-webrtc
//
//  Created by Injoit on 2/25/19.
//  Copyright © 2019 Quickblox. All rights reserved.
//

#import "BaseItemModel.h"
#import "SettingCell.h"

@implementation BaseItemModel

- (Class)viewClass {
    
    return [SettingCell class];
}

@end

File: sample-videochat-webrtc-swift/sample-videochat-webrtc-swift/Components/Settings/BaseSettingsViewController/BaseItemModel.swift
//
//  BaseItemModel.swift
//  sample-videochat-webrtc-swift
//
//  Created by Injoit on 12/11/18.
//  Copyright © 2018 QuickBlox. All rights reserved.
//

import UIKit

class BaseItemModel {
    //MARK: - Properties
    var title = ""
    var data: Any?
    
    //MARK: - Public Methods
    func viewClass() -> AnyClass {
        return SettingCell.self
    }
}
